index.html

```diff
@@ -104,28 +100,27 @@ <h3>Some Capstones</h3>
 </section>
 
 <address>  
-      <p>phone: 0123/3210123</p>
-      <p><a href="mailto:">mailto: newfish@water.com</a></p>
-  </address>
+      <div><img src="images/mail_black_24dp.svg" alt="" class="material-icon"><mailto:">newfish@inthepool.com</a></div>
+      <div><img src="images/phone_black_24dp.svg" alt="" class="material-icon">0123/3210123</div>
+</address>
 
 <section id="private">
-<h3>Special Support</h3>
-<div class="piccontainer">
-  <div class="picstyle">
-  <figure><img src="images/Schreibtischkatze2.jpg" alt="my cat lying on my computer desk" width="400" height="300"></figure>
-  </div>
-  <div class="Mimidescribe"><p>Mimi, beloved cat, with a little handicap after intervertebral disc surgery, 
-    but still marvellous princess attitude</p></div>
-  <div class="picstyle">
-  <figure><img src="images/Schreibtischkatze1.jpg" alt="my cat sitting on my computer desk" width="400" height="650"></figure>
+<h3>My Private Special Support</h3>
+<div class="mimicontainer">
+  <div>
```

Comment:
depends what you want to do ;) in my opinion there are to many `divs`...you could even delete all `divs` in this case and use the section as a container or at least delete just one div!? you have already an `div class="mimicontainer"`

Reply:
let me know what you think ;)

Reply:
i deleted one div so far ... and nothing happened lol so you were right 👍
i want to give the picture a frame and a nice styling of the figcaption in the wider part of the frame at the bottom similar a look of an old polaroid foto - i will try this next, probably i will start new thinking at this point also without the other div